internal/services/cdn/cdn_frontdoor_custom_domain_resource.go

```diff
@@ -120,6 +118,20 @@ func resourceCdnFrontDoorCustomDomain() *pluginsdk.Resource {
 		},
 	}
 
+	if !features.FivePointOhBeta() {
+		tlsElem := resource.Schema["tls"].Elem.(*pluginsdk.Resource)
+		tlsElem.Schema["minimum_tls_version"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Default:  string(cdn.AfdMinimumTLSVersionTLS12),
+			ValidateFunc: validation.StringInSlice([]string{
+				string(cdn.AfdMinimumTLSVersionTLS12),
+				string(cdn.AfdMinimumTLSVersionTLS10),
+			}, false),
+		}
+		resource.Schema["tls"].Elem = tlsElem
```

Comment:
One last suggestion, can we please condense this to
```suggestion
		resource.Schema["tls"].Elem.(*pluginsdk.Resource).Schema["minimum_tls_version"] = &pluginsdk.Schema{
			Type:     pluginsdk.TypeString,
			Optional: true,
			Default:  string(cdn.AfdMinimumTLSVersionTLS12),
			ValidateFunc: validation.StringInSlice([]string{
				string(cdn.AfdMinimumTLSVersionTLS12),
				string(cdn.AfdMinimumTLSVersionTLS10),
			}, false),
		}
```